App1/App1.Android/App1.Android.csproj

```diff
@@ -141,4 +144,4 @@
     </AndroidAsset>
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-</Project>
+</Project>
```

Comment:
Do you mind adding back the newlines that were removed in these csproj files?

Reply:
Ok..... Why?

Reply:
Done.

Reply:
To avoid this:

![image](https://user-images.githubusercontent.com/6355370/83931089-a6bed280-a74f-11ea-9c3a-3cf2ba30e6aa.png)

As to why: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline